### DISCUSSION TITLE
What’s your most performing dub.sh link?

### DISCUSSION THREAD

**Discussion by fmerian** [dubinc/dub#169] [Show and tell]
Happy Friday ☀️

I hit 100 clicks for a link to a HN thread that I recently tweeted [here](https://twitter.com/fmerian/status/1640405208567103513?s=20).

Curiosity -- what’s your most performing dub.sh link?

**Rules:**

- must be a dub.sh link

**Comment by steven-tey** [comment]
Great question! Technically my highest one is `/github` but that doesn't really count since it's the demo link haha

So I guess I'll go with the 2nd highest one – `/tweepcred`, which has 9K clicks thanks to [this tweet](https://twitter.com/steventey/status/1641892389160894464).

![CleanShot 2023-04-28 at 18 50 03](https://user-images.githubusercontent.com/28986134/235271028-f33acf75-2fc6-41c4-b4e8-3032deeb1b5e.png)

**Reply by fmerian** [reply]
wow, impressive! overall, my most performing Dub link hits 3.5K clicks

**Reply by matytyma** [reply]
I've even botted that GitHub link :smile:
